utils/fees.py

```diff
@@ -86,18 +86,21 @@ def convertStablestoLP(stable_lp_maker_v3, w3, acct):
         removeLiquidity = [
             USDC_USDT_BASE_POOL,
         ]
-        # Convert fees from nUSD to USDC
+        # Convert nUSD, USN to USDC
         swaps = [
             NUSD_USDC_USDT_META_DEPOSIT_POOL,
+            USDC_USDT_USN_BASE_POOL,
```

Comment:
Should `auUSDC/auUSDT` pool be added here, too?

Reply:
naa cannot convert those stables into USDC/USDT right now